pkgs/data/themes/marwaita/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, fetchFromGitHub
+, gdk-pixbuf
+, gtk-engine-murrine
+, gtk_engines
+, librsvg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "marwaita";
+  version = "2020-07-01";
```

Comment:
`"unstable-2020-07-01"`, since this isn't an actual version

Reply:
As far as I know adding _unstable_ to the name is not mandatory. There are some discussions on how it should be done (but I am failing to find the link now). It would be part of _pname_ or _version_? It should appear before or after the date? I do not like it written just before the date because it could be understood as part of _pname_, while it really is not (or is it?). I prefer not using _unstable_ on packages without an explicit need to distinguish from stable versions simultaneously available in nixpkgs.

Reply:
I don't really have a horse in this race, but the [guideline](https://nixos.org/nixpkgs/manual/#sec-package-naming) is pretty clear:

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

That is, `pname` should have an "-unstable" suffix. I can see at least one advantage: It makes it possible to distinguish between packages that actually name their releases after dates and packages without releases where we just decided to ship some commit.

Reply:
There are plenty of discussions regarding this use of _unstable_. In particular [this comment](https://github.com/NixOS/nixpkgs/pull/88023#issuecomment-629870275) express my thinking about it.

Some links:
https://github.com/NixOS/nixpkgs/pull/88023
https://github.com/NixOS/nixpkgs/issues/68518
https://github.com/NixOS/nixpkgs/pull/86430

Reply:
Thanks for linking those! As I said, I do think it makes *some* sense, but I don't really have a horse in the race. The PR looks fine otherwise, since you have merge permission yourself I'll let you make the final call.